portality/api/v1/discovery.py

```diff
@@ -89,7 +89,10 @@ def colon_escaper(q):
         # colons after keywords are: first one and every first one after & or |
         parts = q.split(":")
         for i in range(1, len(parts)-1):
-            if '&' not in parts[i] and '|' not in parts[i] and not parts[i].endswith('\\'):
+            if '&' not in parts[i] \
```

Comment:
we've lost the check for `'|'` here - is that intentional? (the comment above has remained the same)